src/sweeper/sweepers/duplicates.py

```diff
@@ -7,52 +7,54 @@
 
 import arcpy
 import re
+import os
 from xxhash import xxh64
 
-
 class DuplicateTest(object):
-    def __init__(self):
+    def __init__(self, workspace, table_name):
+        self.report = {}
+        self.workspace = workspace
+        self.table_name = table_name
 
 
-    def sweep(self, fc):
+    def sweep(self):
 
         digestDict = {}
-        duplicateDict = {}
-        dupePrintLst = []
+
         digTrim = re.compile(r'(\d+\.\d{2})(\d+)')
 
-        shps = ['Shape', 'SHAPE']
+        arcpy.env.workspace = self.workspace
 
-        fields = [f.name for f in arcpy.ListFields(fc) if f.name not in shps]
+        omitFlds = ['Shape', 'SHAPE', 'GLOBALID']
```

Comment:
we have a style guide to use `snake_case` for python code. `omitFlds` would be better expressed as `omit_fields`. I'd love to see you snake case all of your variable names. There is an [vs code extension](https://github.com/wmaurer/vscode-change-case) that can help but it's pretty simple if you use find and replace or [multiple cursors](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_multi-cursor-selection).

Reply:
Ok. I'll talk with Erik, maybe there are some similar variable names that we can standardize.

Reply:
i don't mind what you and erik call things. There is no need to standardize on variable names. As long as they are `snake_cased` for readability and consistency we're all good.